Epsilon.Host.Frontend/src/components/TopNavigation.vue

```diff
@@ -90,18 +88,18 @@ const avatarImage = computed(() => {
     }
 }
 
-.avatar {
-    &-image {
-        min-width: 3rem;
-        min-height: 3rem;
-        max-width: 3rem;
-        max-height: 3rem;
-        aspect-ratio: 1/1;
-        border: none;
-        border-radius: 3rem;
-        margin-right: 1rem;
-        overflow: hidden;
-        background-color: #fff;
-    }
-}
+//.avatar {
+//    &-image {
+//        min-width: 3rem;
+//        min-height: 3rem;
+//        max-width: 3rem;
+//        max-height: 3rem;
+//        aspect-ratio: 1/1;
+//        border: none;
+//        border-radius: 3rem;
+//        margin-right: 1rem;
+//        overflow: hidden;
+//        background-color: #fff;
+//    }
+//}
```

Comment:
Commented code, also why remove this?

Reply:
Because it was determined that getting pictures from canvas was a huge bottleneck in loading times.

Reply:
No need to remove the avatar styling, just use a default character for now and put the feature on the roadmap

Reply:
I get it. But is is in the git history. 
And especially for CSS the less is in the project the better. 
the less stuff it can mess up.